src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -138,16 +138,16 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.positional('source_location', help="The local source code directory path (e.g., './src') or the URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') or a remote tarball (e.g., 'http://server/context.tar.gz'). If '/dev/null' is specified, the value will be set to None and ignored.", completer=FilesCompleter())
         c.argument('file', options_list=['--file', '-f'], help="The task template/definition file path relative to the source context. It can be '-' to pipe a file from the standard input.")
         c.argument('values', help="The task values file path relative to the source context.")
-        c.argument('set_value', options_list=['--set'], help="Value in 'name[=value]' format.", action='append', validator=validate_set)
-        c.argument('set_secret', help="Secret value in 'name[=value]' format.", action='append', validator=validate_set_secret)
+        c.argument('set_value', options_list=['--set'], help="Value in 'name[=value]' format. Multiple --set can be fed.", action='append', validator=validate_set)
```

Comment:
Actually is this something we can change at the CLI level, i.e., if the parameter supports `append` the help message should mention multiple can be fed, etc.?
/cc @tjprescott

Reply:
Possible, though not likely to get development attention in the near future. Particularly with key value pairs it is much more common to do what `--tags` does, accepting a space-separated list of key value pairs:
This: `--tags a=b c=d`
Instead of: `--tags a=b --tags c=d`

For places where append is currently used (and must be due to the complexity) the ability to specify multiple instances is called out in the long summary.